pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -142,7 +150,7 @@ func parseKeys(packedKeys string) (private, public interface{}) {
 }
 
 func encodeVCToJSON(vcBytes []byte) {
-	credential, err := verifiable.NewCredential(vcBytes)
+	credential, err := verifiable.NewCredential(vcBytes, verifiable.WithNoCustomSchemaCheck())
```

Comment:
Is there an issue?

Reply:
In the Test Suite, there is a suite that checks Credential Schemas of Credential. They use fake URLs and `NewCredential()` because of that. Using `WithNoCustomSchemaCheck()`, we disable loading of custom schemas and thus get rid of the issue.